src/Message/PurchaseRequest.php

```diff
@@ -41,11 +41,57 @@ public function setEntranceCode($value)
         return $this->setParameter('entranceCode', $value);
     }
 
+    public function getMakeInvoice()
+    {
+        return $this->getParameter('makeInvoice') ?: 'true';
+    }
+
+    public function setMakeInvoice($value)
+    {
+        return $this->setParameter('makeInvoice', $value);
+    }
+
+    public function getMailInvoice()
+    {
+        return $this->getParameter('mailInvoice') ?: 'true';
```

Comment:
I'd rather stick to the defaults, so sending `null` and let Sisow decide the default.